bib/cmd/bootc-image-builder/image.go

```diff
@@ -42,6 +42,9 @@ type ManifestConfig struct {
 
 	// TLSVerify specifies whether HTTPS and a valid TLS certificate are required
 	TLSVerify bool
+
+	// Use a local container from the host rather than a repository
```

Comment:
I don't think BIB is using container but container images?  Should this be
Use a local container image ...